src/Components/Facility/AssetCreate.tsx

```diff
@@ -539,17 +549,26 @@ const AssetCreate = (props: AssetProps) => {
                     <div>
                       <label htmlFor="asset-class">Asset Class</label>
                       <div className="mt-2">
-                        <SelectMenu
+                        <SelectMenuV2
+                          required
                           options={[
-                            { title: "Not Applicable", value: undefined },
+                            { title: "Select", value: "NONE" },
```

Comment:
SelectMenuV2 has a placeholder prop, use that to achieve this. Adding a default option in the list of options makes it an actual valid value in the perspective of the SelectMenuV2 component.